src/sql/engine/px/ob_dfo.h

```diff
@@ -182,6 +182,26 @@ struct ObP2PDhMapInfo
   TO_STRING_KV(K_(p2p_sequence_ids), K_(target_addrs));
 };
 
+struct ObQcRecordData {
```

Comment:
1. Why do we need this separate data structure？Can we just embed this in SqcMeta？
2. Make sure the px worker can access sqc meta. (usually px worker access task meta data structure)

Reply:
If we do need this one. Can we move more info into this.

QcRecordData is not so meaningful.

What about ObQcDebugInfo / ObQcDiagnosisData / ObQcDebugData / ....?  Discuss this with your boss.

Reply:
> If we do need this one. Can we move more info into this.
> 
> QcRecordData is not so meaningful.
> 
> What about ObQcDebugInfo / ObQcDiagnosisData / ObQcDebugData / ....? Discuss this with your boss.

1. Fields in this structure are used only to monitor QC and will not be used in other places.
2. I have renamed it to `ObQCRecordMonitorInfo`~